packages/documentation/src/stories/components/forms/search-input/search-input.snaphot.stories.ts

```diff
@@ -0,0 +1,71 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta from './search-input.stories';
+import { html } from 'lit';
+import { getCombinations } from '@/utils/inputComponentsGetCombinations';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+  render: renderSearchInputSnapshot,
+};
+
+function renderSearchInputSnapshot(_args: Args, context: StoryContext) {
+  const combinations = [
```

Comment:
I'm not sure those are actually used. The labels are not actually displayed on the snapshot, you might be able to simplify this file a bit.